spec/system/admin/order_cycles/edit_spec.rb

```diff
@@ -41,11 +41,8 @@
 
         # change date range field value
         find('#order_cycle_orders_close_at').click
-        within(".flatpickr-calendar.open") do
-          expect(page).to have_selector '.shortcut-buttons-flatpickr-buttons'
-          select_datetime_from_datepicker Time.zone.parse("2024-03-30 00:00")
-          find("button", text: "Close").click
-        end
+        select_datetime_from_datepicker Time.zone.parse("2024-03-30 00:00")
```

Comment:
This little feature you added in `datepicker_time` would be handy for `select_datetime_from_datepicker` also:
```
datetime = Time.zone.parse(datetime) if datetime.is_a? String
```

Then we can be even  more succinct:
```suggestion
        select_datetime_from_datepicker "2024-03-30 00:00"
```

Reply:
Good idea. But I think I spent enough time on cleaning up specs.